doc/glossary.rst

```diff
@@ -1696,9 +1696,16 @@ functions or non-estimator constructors.
         objects and avoid common pitfalls, you may refer to :ref:`randomness`.
 
     ``scoring``
-        Specifies the score function to be maximized (usually by :ref:`cross
-        validation <cross_validation>`), or -- in some cases -- multiple score
-        functions to be reported. The score function can be a string accepted
+        Depending on the object, can specify:
+
+        * the score function to be maximized (usually by
+          :ref:`cross validation <cross_validation>`),
+        * in some cases, the multiple score functions to be reported,
+        * in a few specific cases, the score function to be used to check early
+          stopping, or
```

Comment:
Given this is a list, I think we can remove the "in some cases" part:

```suggestion
        * multiple score functions to be reported,
        * the score function to be used to check early stopping, or
```

Reply:
Done thank you!